src/link/ping1dsimulationlink.cpp

```diff
@@ -35,12 +36,12 @@ void Ping1DSimulationLink::randomUpdate()
     for (int i = 0; i < numPoints; i++) {
         float point;
         if (i < stop1) {
-            point = 0.1 * (qrand() % 256);
+            point = 0.1 * QRandomGenerator::global()->bounded(256);
```

Comment:
why 256? couldn't you do `QRandomGenerator::global()->bounded(25.6)`?

Reply:
From the doc: 
`Generates one random double in the range between 0 (inclusive) and highest (exclusive).`
Probably you'll catch it better as an [0, 256) interval that is the same as [0, 255].

Reply:
@Williangalvani let me know if this resolve the conversation

Reply:
The code says the function is equivalent to `generateDouble() * highest`, so I thought you just move that 0.1 inside. I understand that double and integer generation will yeld different results, but I thought it didn't matter as much here.

I'm not really concerned about this, though, marking as resolved.